playbooks/common/openshift-master/config.yml

```diff
@@ -38,6 +38,11 @@
           public_hostname: "{{ openshift_public_hostname | default(None) }}"
           public_ip: "{{ openshift_public_ip | default(None) }}"
           deployment_type: "{{ openshift_deployment_type }}"
+      - role: nfs
+        local_facts:
+          host: "{{ openshift_nfs_host | default(None) if openshift_nfs_host is defined else groups.oo_nfs_to_config.0 if groups.oo_nfs_to_config | length > 0 else None }}"
```

Comment:
Why not move some of this to the play vars?

In fact, it looks like the set_fact in pre_tasks can be moved to the play vars as well.